subprojects/docs/build.gradle

```diff
@@ -196,6 +196,9 @@ samples {
         }
     }
 
+    templates.each {
+        println("Template: ${it.name} ${it.target.get()} ${it.getSourceDirectory().getAsFile().get()} ${it.getTemplateDirectory().getAsFile().get()}")
+    }
```

Comment:
I assume this is not supposed to get merged in.

Reply:
Yes, thanks for pointing out. Removed.